wideq/ac.py

```diff
@@ -7,6 +7,28 @@
 from .core import FailedRequestError
 
 
+class ACJetMode(enum.Enum):
+    """ To turn on the jet mode. This mode
+    puts your AC into highest cooling or
+    dry or heat mode(for a certain amount of time)
+    depending on what you choose
+
+    This mode Overrides following setting:
+    1. Vertical swing is set to @100
+    2. Temperature gets set to 18 after
+    jet mode turns off
+    3. Fan speed is set to HIGH
+    (@AC_MAIN_WIND_STRENGTH_HIGH_W)
+    after jet mode turns off
```

Comment:
This docstring is wrapped rather narrowly—it can be 72 characters wide to be easier to read.

Reply:
done